compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -252,7 +252,11 @@ object Completion {
       (
            (mode.is(Mode.Term) && sym.isTerm)
         || (mode.is(Mode.Type) && (sym.isType || sym.isStable))
-      )
+      ) &&
+      !sym.isPackageObject &&
```

Comment:
This one makes sense, but I'd rather not add a bunch of special cases for repeated, byname, etc. Instead I suggest adding the `Artifact` flag to these symbols (we should already be filtering out Artifacts because they should never appear in user code).

Reply:
Added the `Artifact` flag